src/NuGet.Core/NuGet.Versioning/VersionRange.cs

```diff
@@ -58,9 +58,10 @@ public VersionRange(VersionRange range, FloatRange floatRange)
         /// <param name="includeMaxVersion">True if maxVersion satisfies the condition.</param>
         /// <param name="floatRange">The floating range subset used to find the best version match.</param>
         /// <param name="originalString">The original string being parsed to this object.</param>
+        /// <param name="isCentral">A flag to signal that the version range is central defined.</param>
         public VersionRange(NuGetVersion minVersion = null, bool includeMinVersion = true, NuGetVersion maxVersion = null,
```

Comment:
IsCentral does not belong here. 
IsCentral describes a package reference version range, not a version range.

Reply:
Agree. Very good point! Removed.